src/components/Messages.js

```diff
@@ -1,12 +1,78 @@
 import React from 'react'
+import {Component} from 'react'
 import { Button } from 'react-bootstrap';
+import ReactDOM from 'react-dom';
 
-function About() {
-    return <div className="content">
-        <h1>Messages page</h1>
-        <Button> Inbox </Button> 
-        <Button> Sent </Button>
-      </div>;
+  class Messages extends React.Component {
+    constructor(props) {
+      super(props);
+      this.handleInboxClick = this.handleInboxClick.bind(this);
+      this.handleBackClick = this.handleBackClick.bind(this);
+      this.state = {isLoggedIn: false};
+    }
+  
+    handleInboxClick() {
+      this.setState({isLoggedIn: true});
```

Comment:
Why is this called "isLoggedIn" when its purpose is only to open the inbox?  Wouldn't "inboxOpen" be a better name for clarity?